src/targets/gpu/include/migraphx/gpu/miopen.hpp

```diff
@@ -75,21 +75,31 @@ inline tensor_descriptor make_tensor(const migraphx::shape& os, bool pack = fals
     return t;
 }
 
+inline void insert_attribute_1d_dims(std::vector<int>& padding,
+                                     std::vector<int>& stride,
+                                     std::vector<int>& dilation)
+{
+    padding.insert(padding.begin(), 0);
+    stride.insert(stride.begin(), 1);
+    dilation.insert(dilation.begin(), 1);
```

Comment:
Can we just default initialize these and then copy backwards into the vector?

Reply:
changed